classes/property_list.php

```diff
@@ -173,6 +173,21 @@ public function delete_element(string $key) {
         }, $this->cfpropertylist->getValue());
     }
 
+    /**
+     * Helper function to either set or update a CF type value to the plist.
+     *
+     * @param string $key
+     * @param CFType $input
+     */
+    public function set_or_update_value(string $key, CFType $input) {
```

Comment:
Would be good to add some tests for this method

Reply:
I've now updated property_list_test.php and added test_set_or_update_value.